src/System.Private.CoreLib/src/System/Runtime/InteropServices/InteropExtensions.cs

```diff
@@ -357,12 +357,13 @@ public static uint RuntimeHandleCompareExchangeVariableType(IntPtr handle, uint
         {
             return RuntimeImports.RhHandleCompareExchangeVariableType(handle, oldType, newType);
         }
-
-        public static void SetExceptionErrorCode(Exception exception, int hr)
-        {
-            exception.SetErrorCode(hr);
-        }
-
+        
+#if PROJECTN
+         public static void SetExceptionErrorCode(Exception exception, int hr)
+         {
+             // TODO: delete
```

Comment:
This needs to be actually implemented. The stuff will break otherwise.

Reply:
Fixed